documentation/design-docs/ipc-protocol.md

```diff
@@ -143,66 +143,80 @@ Packets will contain a header and 0 or one payload.  The payload will completely
     <th>22</th>
     <th>23</th>
     <th>24</th>
-    <th>25</th>
-    <th>26</th>
     <th>...</th>
     <th>size - 1 </th>
     <th>size</th>
   </tr>
   <tr>
-    <td colspan="22">header</td>
+    <td colspan="20">header</td>
     <td colspan="7">payload</td>
   </tr>
   <tr>
-    <td colspan="16">magic</td>
+    <td colspan="14">magic</td>
     <td colspan="2">size</td>
     <td colspan="2">command</td>
     <td colspan="2">reserved</td>
     <td colspan="7">payload</td>
   </tr>
 </table>
 
-The simplest of command/reply packets will only contain a header and therefore be only 22 bytes long.
+The simplest Diagnostic IPC Message will contain only a header and therefore only be 20 bytes long.
+
+### Commands
 
-### Commands Sets
+Commands are a combination of a `command_set` (most significant byte) and a `command_id` (least significant byte).  Commands are application specific and don't affect the specification of this protocol beyond their encoding in the header.
```

Comment:
I think the list of commands, their ids, arguments and return data are all in scope. We can divide the protocol into layers OS transport/message format/command payloads, but all three layers will need to be covered in order for external devs to use the protocol.

Reply:
Would it be better to split this into multiple docs or keep it all in here?

Reply:
My suggestion would be to leave it together, but as long as a dev can find all the info easily I don't think it matters that much.